nl/mail/chrome/messenger/updates/pbUpdates.properties

```diff
@@ -1 +1 @@
-UpToDate=Postbox %S is currently the newest version available.
+UpToDate=Postbox %S is op dit moment de nieuwste versie die beschikbaar is.
```

Comment:
@kikikir What should the corrected line be?  The way this displays in English is:

Postbox 4.0.8 is currently the newest version available.